src/routes/UserRouter.js

```diff
@@ -91,6 +89,49 @@ export default class UserRouter {
         });
     }
 
+    verifyUser(req: $Request, res: $Response): void {
+        const { body } = req;
+        const email = body.email;
+        const href = body.href;
+
```

Comment:
Check if link is legit -> check if link conforms to email -> update `verified` flag to true